--- open-vm-tools-10.1.10-6082533/lib/nicInfo/nicInfoPosix.c~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/lib/nicInfo/nicInfoPosix.c	2017-09-27 01:33:33.633701399 +0000
@@ -409,7 +409,11 @@
    return ipstr;
 }
 #else
-#   error GuestInfoGetPrimaryIP needed for this platform
+char *
+GuestInfoGetPrimaryIP(void)
+{
+   return strdup("127.0.0.1");
+}
 #endif
 #endif
 
--- open-vm-tools-10.1.10-6082533/configure~	2017-07-28 22:19:58.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/configure	2017-09-27 01:35:48.584152356 +0000
@@ -25443,7 +25443,6 @@
 
 ### General flags / actions
 CFLAGS="$CFLAGS -Wall"
-CFLAGS="$CFLAGS -Werror"
 
 # -Wno-unknown-pragmas is due to gcc not understanding '#pragma ident'
 # in Xlib.h on OpenSolaris.
