--- open-vm-tools-10.1.10-6082533/lib/guestApp/guestApp.c~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/lib/guestApp/guestApp.c	2017-09-30 15:39:24.435649911 +0000
@@ -63,7 +63,7 @@
 #elif defined __APPLE__
 #   define GUESTAPP_TOOLS_INSTALL_PATH "/Library/Application Support/VMware Tools"
 #else
-#   define GUESTAPP_TOOLS_INSTALL_PATH "/etc/vmware-tools"
+#   define GUESTAPP_TOOLS_INSTALL_PATH "/usr/share/defaults/open-vm-tools/vmware-tools"
 #endif
 
 #if defined _WIN32
--- open-vm-tools-10.1.10-6082533/vgauth/service/Makefile.in~	2017-07-28 22:20:07.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/vgauth/service/Makefile.in	2017-09-30 15:40:33.092421883 +0000
@@ -489,7 +489,7 @@
 	../common/audit.c ../common/certverify.c ../common/i18n.c \
 	../common/prefs.c ../common/usercheck.c ../common/VGAuthLog.c \
 	../common/VGAuthUtil.c
-VGAuthServicedir = /etc/vmware-tools/vgauth/schemas
+VGAuthServicedir = /usr/share/defaults/open-vm-tools/vmware-tools/vgauth/schemas
 VGAuthService_SCRIPTS = ../serviceImpl/schemas/datatypes.dtd \
 	../serviceImpl/schemas/saml-schema-assertion-2.0.xsd \
 	../serviceImpl/schemas/xenc-schema.xsd \
@@ -1262,7 +1262,7 @@
 # Message catalogs.
 install-data-hook:
 	@INSTVMSG@ VGAuthService $(srcdir)/l10n $(DESTDIR)$(datadir)
-	cat vgauth.conf | sed -e"s!@@VGAUTHSCHEMADIR@@!$(VGAuthServicedir)!" > $(DESTDIR)/etc/vmware-tools/vgauth.conf
+	cat vgauth.conf | sed -e"s!@@VGAUTHSCHEMADIR@@!$(VGAuthServicedir)!" > $(DESTDIR)/usr/share/defaults/open-vm-tools/vmware-tools/vgauth.conf
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
--- open-vm-tools-10.1.10-6082533/vgauth/service/Makefile.am~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/vgauth/service/Makefile.am	2017-09-30 15:40:50.490421313 +0000
@@ -47,7 +47,7 @@
 VGAuthService_SOURCES += ../common/VGAuthLog.c
 VGAuthService_SOURCES += ../common/VGAuthUtil.c
 
-VGAuthServicedir = /etc/vmware-tools/vgauth/schemas
+VGAuthServicedir = /usr/share/defaults/open-vm-tools/vmware-tools/vgauth/schemas
 VGAuthService_SCRIPTS =
 VGAuthService_SCRIPTS += ../serviceImpl/schemas/datatypes.dtd
 VGAuthService_SCRIPTS += ../serviceImpl/schemas/saml-schema-assertion-2.0.xsd
@@ -101,4 +101,4 @@
 # Message catalogs.
 install-data-hook:
 	@INSTVMSG@ VGAuthService $(srcdir)/l10n $(DESTDIR)$(datadir)
-	cat vgauth.conf | sed -e"s!@@VGAUTHSCHEMADIR@@!$(VGAuthServicedir)!" > $(DESTDIR)/etc/vmware-tools/vgauth.conf
+	cat vgauth.conf | sed -e"s!@@VGAUTHSCHEMADIR@@!$(VGAuthServicedir)!" > $(DESTDIR)/usr/share/defaults/open-vm-tools/vmware-tools/vgauth.conf
--- open-vm-tools-10.1.10-6082533/vgauth/common/prefs.h~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/vgauth/common/prefs.h	2017-09-30 15:41:11.311819114 +0000
@@ -64,7 +64,7 @@
 // fallback value if registry isn't set
 #define VGAUTH_PREF_CONFIG_FILENAME "c:\\Program Files\\VMware\\VMware Tools\\vgauth.conf"
 #else
-#define VGAUTH_PREF_CONFIG_FILENAME "/etc/vmware-tools/vgauth.conf"
+#define VGAUTH_PREF_CONFIG_FILENAME "/usr/share/defaults/open-vm-tools/vmware-tools/vgauth.conf"
 // XXX temp til installer tweaks its location
 #define VGAUTH_PREF_CONFIG_FILENAME_OLD "/etc/vmware/vgauth.conf"
 #endif
@@ -101,7 +101,7 @@
 auditSuccessEvents=true
 
 [localization]
-msgCatalog = /etc/vmware-tools/vgauth/messages
+msgCatalog = /usr/share/defaults/open-vm-tools/vmware-tools/vgauth/messages
 # EOF
  @endverbatim
  * See http://developer.gnome.org/glib/2.28/glib-Key-value-file-parser.html#glib-Key-value-file-parser.description
--- open-vm-tools-10.1.10-6082533/vmware-user-suid-wrapper/wrapper.h~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/vmware-user-suid-wrapper/wrapper.h	2017-09-30 15:41:28.598299162 +0000
@@ -43,7 +43,7 @@
  * in the latter.
  */
 #ifdef USES_LOCATIONS_DB
-#   define LOCATIONS_PATH       "/etc/vmware-tools/locations"
+#   define LOCATIONS_PATH       "/usr/share/defaults/open-vm-tools/vmware-tools/locations"
 
 /*
  * Locations DB query selector.  Values in this enum are used as array
--- open-vm-tools-10.1.10-6082533/scripts/Makefile.in~	2017-07-28 22:20:06.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/scripts/Makefile.in	2017-09-30 15:41:41.864298727 +0000
@@ -355,7 +355,7 @@
 top_srcdir = @top_srcdir@
 defaultscripts = poweron-vm-default poweroff-vm-default \
 	suspend-vm-default resume-vm-default
-confdir = /etc/vmware-tools
+confdir = /usr/share/defaults/open-vm-tools/vmware-tools
 conf_SCRIPTS = ./common/vm-support ./common/statechange.subr \
 	$(am__append_1) $(defaultscripts)
 vmwsrcdir = $(confdir)/scripts/vmware
--- open-vm-tools-10.1.10-6082533/scripts/common/vm-support~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/scripts/common/vm-support	2017-09-30 15:42:01.374625245 +0000
@@ -327,7 +327,7 @@
 # Common stuff that we gather for all OSes.
 runcmd "/tmp/vm-support-version.txt" echo vm-support version: $VER
 
-addfiles /etc/vmware-tools
+addfiles /usr/share/defaults/open-vm-tools/vmware-tools
 addfiles /var/log/boot*
 addfiles /var/log/secure*
 addfiles /var/log/messages*
--- open-vm-tools-10.1.10-6082533/scripts/common/statechange.sh~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/scripts/common/statechange.sh	2017-09-30 15:42:34.285055217 +0000
@@ -31,7 +31,7 @@
 #
 # Handy reference/shorthand used in this doc/scripts:
 #    TOOLS_CONFDIR ::= Depends on platform and installation settings.  Likely
-#                      "/etc/vmware-tools" or
+#                      "/usr/share/defaults/open-vm-tools/vmware-tools" or
 #                      "/Library/Application Support/VMware Tools"
 #    powerOp       ::= One of "poweron-vm", "poweroff-vm", "suspend-vm", and
 #                      "resume-vm".
--- open-vm-tools-10.1.10-6082533/scripts/Makefile.am~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/scripts/Makefile.am	2017-09-30 15:42:45.243054858 +0000
@@ -3,7 +3,7 @@
 defaultscripts += suspend-vm-default
 defaultscripts += resume-vm-default
 
-confdir = /etc/vmware-tools
+confdir = /usr/share/defaults/open-vm-tools/vmware-tools
 
 conf_SCRIPTS = ./common/vm-support
 conf_SCRIPTS += ./common/statechange.subr
--- open-vm-tools-10.1.10-6082533/common-agent/Cpp/ManagementAgent/Subsystems/MaIntegration/src/CConfigEnv.cpp~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/common-agent/Cpp/ManagementAgent/Subsystems/MaIntegration/src/CConfigEnv.cpp	2017-09-30 15:43:02.172054303 +0000
@@ -62,7 +62,7 @@
 		CEnvironmentUtils::readEnvironmentVar("ProgramData", programData);
 		guestProxyDir = FileSystemUtils::buildPath(programData, "VMware", "VMware Tools", "GuestProxyData");
 #else
-		guestProxyDir = "/etc/vmware-tools/GuestProxyData";
+		guestProxyDir = "/usr/share/defaults/open-vm-tools/vmware-tools/GuestProxyData";
 #endif
 
 		_vcidPath = FileSystemUtils::buildPath(guestProxyDir, "VmVcUuid", "vm.vc.uuid");
--- open-vm-tools-10.1.10-6082533/lib/misc/codeset.c~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/lib/misc/codeset.c	2017-09-30 15:43:21.621589927 +0000
@@ -100,7 +100,7 @@
 #   define POSIX_ICU_DIR DEFAULT_LIBDIRECTORY
 #elif !defined _WIN32
 #   if defined(VMX86_TOOLS)
-#      define POSIX_ICU_DIR "/etc/vmware-tools"
+#      define POSIX_ICU_DIR "/usr/share/defaults/open-vm-tools/vmware-tools"
 #   else
 #      define POSIX_ICU_DIR "/etc/vmware"
 #   endif
--- open-vm-tools-10.1.10-6082533/services/vmtoolsd/Makefile.in~	2017-07-28 22:20:07.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/services/vmtoolsd/Makefile.in	2017-09-30 15:43:34.812589495 +0000
@@ -954,10 +954,10 @@
 	@INSTVMSG@ vmtoolsd $(srcdir)/l10n $(DESTDIR)$(datadir)
 
 install-exec-hook:
-	$(INSTALL) -d $(DESTDIR)/etc/vmware-tools
+	$(INSTALL) -d $(DESTDIR)/usr/share/defaults/open-vm-tools/vmware-tools
 
 uninstall-hook:
-	rm -rf $(DESTDIR)/etc/vmware-tools
+	rm -rf $(DESTDIR)/usr/share/defaults/open-vm-tools/vmware-tools
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
--- open-vm-tools-10.1.10-6082533/services/vmtoolsd/Makefile.am~	2017-07-28 22:19:20.000000000 +0000
+++ open-vm-tools-10.1.10-6082533/services/vmtoolsd/Makefile.am	2017-09-30 15:43:47.778776969 +0000
@@ -83,8 +83,8 @@
 	@INSTVMSG@ vmtoolsd $(srcdir)/l10n $(DESTDIR)$(datadir)
 
 install-exec-hook:
-	$(INSTALL) -d $(DESTDIR)/etc/vmware-tools
+	$(INSTALL) -d $(DESTDIR)/usr/share/defaults/open-vm-tools/vmware-tools
 
 uninstall-hook:
-	rm -rf $(DESTDIR)/etc/vmware-tools
+	rm -rf $(DESTDIR)/usr/share/defaults/open-vm-tools/vmware-tools
 
